forward/fwd.go

```diff
@@ -318,9 +316,13 @@ func (f *websocketForwarder) copyRequest(req *http.Request, u *url.URL) (outReq
 		outReq.URL.Scheme = "ws"
 	}
 
+	requestURI, err := url.ParseRequestURI(outReq.RequestURI)
+	if err == nil {
```

Comment:
To follow the best practice of reducing variable scopes to the least minimum. Just like you wouldn't reach out for global variables unless you needed too.